tox.ini

```diff
@@ -1,55 +1,89 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/zope-product
 [tox]
+minversion = 3.18
 envlist =
-   py27,
-   py35,
-   py36,
-   py37,
-   py38,
-   flake8,
-   coverage,
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    coverage
 
 [testenv]
+skip_install = true
+deps =
+    zc.buildout
 commands_pre =
-    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test_with_gs
+    py27,py35: {envbindir}/buildout -nc {toxinidir}/buildout4.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test test_with_gs
+    !py27-!py35: {envbindir}/buildout -nc {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test test_with_gs
 commands =
     {envbindir}/test {posargs:-cv}
     {envbindir}/test_with_gs {posargs:-cv}
-deps =
-    zc.buildout
-skip_install = true
 
-[testenv:coverage]
+[testenv:lint]
 basepython = python3
-commands =
-    coverage run {envbindir}/test_with_gs {posargs:-cv}
-    coverage html
-    coverage xml
-    coverage report
-deps =
-    {[testenv]deps}
-    coverage
-
-[testenv:flake8]
-basepython = python3.6
+allowlist_externals =
+    mkdir
 commands_pre =
     mkdir -p {toxinidir}/parts/flake8
 commands =
-    isort --check-only --diff --recursive {toxinidir}/src setup.py
-    - flake8 --format=html --doctests src setup.py
-    flake8 --doctests src setup.py
+    - isort --check-only --diff {toxinidir}/src setup.py
```

Comment:
Where does this and the following line come from? I cannot deduce those from the zopefoundation/meta changes. Do we need the `-` here?

Reply:
Ah, okay I found the place. https://github.com/zopefoundation/meta/pull/57/files#diff-e9d7c2feed27afda348a43f05d3abbe05a27e0fc9d4a0149c3c6ba848be84e56R33